tests/test_heat.py

```diff
@@ -212,6 +267,12 @@ def test_disconnected_pipe_darcy_weisbach(self):
         """
         Just a sanity check that the head loss constraints for disconnectable
         pipes works with LINEAR as well as LINEARIZED_DW.
+
+        Checks:
+        - That flow is equal between both
```

Comment:
Do you mean: "That the flow is equal for both types of head loss constraint settings" ?

Reply:
done